docs/presets/README.md

```diff
@@ -466,3 +466,60 @@ format = 'via [$symbol]($style)'
 [zig]
 format = 'via [$symbol]($style)'
 ```
+
+## Pure
+
+This preset emulates the look and behavior of [Pure](https://github.com/sindresorhus/pure).
+
+![Screenshot of Pure preset](/presets/pure-prompt.png)
+
+### Configuration
+
+```toml
+format = """
+$username\
+$hostname\
+$directory\
+$git_branch\
+$git_state\
+$git_status\
+$cmd_duration\
+$line_break\
+$python\
+$character"""
+
+[directory]
+style = "blue"
+
+[character]
+success_symbol = "[❯](5)"
+error_symbol = "[❯](red)"
+vicmd_symbol = "[❮](green)"
+
+[git_branch]
+format = "[$branch]($style)"
+style = "242"
+
+[git_status]
+format = "[[$conflicted$deleted$renamed$modified$untracked](218) $ahead_behind$stashed]($style)"
+style = "cyan"
+diverged = "*"
+untracked = "*"
+modified = "*"
+staged = "*"
+renamed = "*"
+deleted = "*"
```

Comment:
This will print multiple `*`-characters if multiple of the conditions for these characters are met. I think you want to do something like `(*$conflicted$deleted$renamed$modified$untracked)` in the above format string and set all these variable to zero-width spaces (empty strings would cause them to be ignored).